lib/base/perfdatavalue.cpp

```diff
@@ -259,6 +259,10 @@ PerfdataValue::Ptr PerfdataValue::Parse(const String& perfdata)
 
 	double value = Convert::ToDouble(tokens[0].SubStr(0, pos));
 
+	if (value == INFINITY || value == - INFINITY) {
```

Comment:
There are special functions for such checks like [`std::isinf()`](https://en.cppreference.com/w/cpp/numeric/math/isinf). A related function like [`std::isfinite()`](https://en.cppreference.com/w/cpp/numeric/math/isfinite) or [`std::isnormal()`](https://en.cppreference.com/w/cpp/numeric/math/isnormal) might be even better suited here as it excludes more edge cases (I haven't checked if subnormal numbers would be a problem, that seems to be the only difference between these two functions).

Reply:
Please excuse my `C`. Those functions do seem more appropriate and sane to use.
I will change that accordingly. Thank you for the tip

Reply:
done